src/Umbraco.Infrastructure.PublishedCache/PublishedMember.cs

```diff
@@ -141,10 +141,6 @@ private void EnsureMemberProperty(List<IPublishedProperty> properties, List<stri
 
         public override string UrlSegment => throw new NotSupportedException();
```

Comment:
This class should exist in Abstractions, it has no ties to nucache and could be re-used for a different implementation.